x-pack/docs/en/security/authentication/configuring-pki-realm.asciidoc

```diff
@@ -179,3 +215,77 @@ NOTE: The PKI realm supports
 alternative to role mapping.
 
 --
+
+[float]
+[role="xpack"]
+[[pki-realm-for-proxied-clients]]
+==== PKI authentication for clients connecting to {kib}
+
+Clients must normally connect directly to {es}, because the {es} PKI realm
+relies on the node's network interface to do the SSL/TLS authentication and to
+also internally forward the client's certificate chain for the realm to be able
+to optionally validate it. If SSL/TLS authenticatication is to be performed
+by {kib}, then the PKI realm must be configured to permit delegating it.
+
+Specifically, when clients presenting X.509 certificates connect to {kib},
+{kib} will perform the SSL/TLS authentication and will forward the client's
+certificate chain, by calling an {es} API, to have them further validated by
+the PKI realms that have been configured for delegation.
+
+To permit authentication delegation for a specific {es} PKI realm, start by
+configuring the realm for the usual case, as detailed in the
+<<pki-realm-for-direct-clients,PKI authentication for clients connecting
+directly to {es}>> section, but make sure that you have explicitly configured
+a `truststore` (or, equivalently `certificate_authorities`) even though it is
+the same trust configuration that you have configured on the network layer.
+Afterwards, simply toggle the `delegation.enabled` realm setting to `true`.
+This realm is now allowed to validate delegated PKI authentication (after
+restarting {es}).
+
+NOTE: PKI authentication delegation requires that
+`xpack.security.authc.token.enabled` be `true`.
+
+NOTE: {kib} also needs to be configured to allow PKI certificate authentication.
+
+A PKI realm with `delegation.enabled` still works unchanged for clients
+connecting directly to {es}. Directly authenticated users and users PKI
+authenticated by delegation to {kib} indistinctly follow the same
```

Comment:
I'm not sure what "indistinctly" means here.  Should it be "indirectly"?

Reply:
it means "both", I replaced it.